services/search/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Use the official Python image.
+# https://hub.docker.com/_/python
+FROM python:3.7-slim
+
+# Copy local code to the container image.
+ENV APP_HOME /app
+WORKDIR $APP_HOME
+COPY . .
+
+# Install production dependencies.
+RUN pip install functions-framework
```

Comment:
add `function-framework` to `requirements.txt` so you save the extra layer and also have an accurate outline of the required dependencies to run this.